test/fixtures/app/main.js

```diff
@@ -0,0 +1,33 @@
+/* eslint no-console: 0 */
```

Comment:
This does not feel like it should live in the fixtures file, which I've always associated more with static (i.e. JSON, CSV, etc.) files.

This feels more like the subject of the test and so I would suggest all files in `/test/fixtures/app/` living somewhere like:
- `test/handlebars/lib/app/` or
- `test/handlebars/utils/app/`